### PR TITLE
CI:  Support package tests for ubuntu-focal and ubuntu-jammy

### PR DESCRIPTION
**Pull request by daipom**

### CODE REVIEW

**Comment by daipom** [comment]
I am checking now if this works correctly.

**Comment by daipom** [comment]
An error occurs in `test/tool/run-test.sh` for ubuntu-jammy

https://github.com/milter-manager/milter-manager/actions/runs/3088131974/jobs/4994315881

**Comment by daipom** [comment]
Should we also add ubuntu-focal and ubuntu-jammy here?

https://github.com/milter-manager/milter-manager/blob/master/package/Rakefile#L67-L71

**Comment by daipom** [comment]
A new error occurs.

https://github.com/milter-manager/milter-manager/actions/runs/3088255226/jobs/4994581855

```
 mkdir -p debian/tmp/usr/lib/python3/
mv debian/tmp-meson/usr/lib/python*/site-packages/	\
   debian/tmp/usr/lib/python3/dist-packages/
mv: cannot stat 'debian/tmp-meson/usr/lib/python*/site-packages/': No such file or directory
make[1]: *** [debian/rules:49: override_dh_auto_install] Error 1
make[1]: Leaving directory '/build/milter-manager-2.1.7'
make: *** [debian/rules:15: binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
debuild: fatal error at line 1182:
dpkg-buildpackage -us -uc -ui failed
```

**Comment by kou** [comment]
> Should we also add ubuntu-focal and ubuntu-jammy here?
>
> https://github.com/milter-manager/milter-manager/blob/master/package/Rakefile#L67-L71

No. We use PPA for Ubuntu.

**Comment by kou** [comment]
https://github.com/milter-manager/milter-manager/actions/runs/3088255226/jobs/4994581855#step:7:6326

> Installing /build/milter-manager-2.1.7/binding/python/gi/overrides/MilterClient/__init__.py to /build/milter-manager-2.1.7/debian/tmp-meson/usr/lib/python3/dist-packages/gi/overrides/MilterClient

`dist-packages` not `site-packages` is used.

```diff
diff --git a/package/debian/python3-milter-client.install b/package/debian/python3-milter-client.install
index 4497f669..a2d85c1d 100644
--- a/package/debian/python3-milter-client.install
+++ b/package/debian/python3-milter-client.install
@@ -1,2 +1,2 @@
-usr/lib/python3/dist-packages/gi/overrides/MilterClient
-usr/lib/python3/dist-packages/milter/client/
+usr/lib/python3/*-packages/gi/overrides/MilterClient
+usr/lib/python3/*-packages/milter/client/
diff --git a/package/debian/python3-milter-core.install b/package/debian/python3-milter-core.install
index 848dd990..678dafd0 100644
--- a/package/debian/python3-milter-core.install
+++ b/package/debian/python3-milter-core.install
@@ -1,3 +1,3 @@
-usr/lib/python3/dist-packages/gi/overrides/MilterCore
-usr/lib/python3/dist-packages/milter/__init__.py
-usr/lib/python3/dist-packages/milter/core/
+usr/lib/python3/*-packages/gi/overrides/MilterCore
+usr/lib/python3/*-packages/milter/__init__.py
+usr/lib/python3/*-packages/milter/core/
diff --git a/package/debian/rules b/package/debian/rules
index fbed4fa1..4d1b727e 100755
--- a/package/debian/rules
+++ b/package/debian/rules
@@ -51,8 +51,8 @@ override_dh_auto_install:
 	mv debian/tmp-meson/usr/lib/$(DEB_HOST_MULTIARCH)/girepository-1.0/	\
 	   debian/tmp/usr/lib/$(DEB_HOST_MULTIARCH)/
 	mkdir -p debian/tmp/usr/lib/python3/
-	mv debian/tmp-meson/usr/lib/python*/site-packages/	\
-	   debian/tmp/usr/lib/python3/dist-packages/
+	mv debian/tmp-meson/usr/lib/python*/*-packages/	\
+	   debian/tmp/usr/lib/python3/
 	mv debian/tmp-meson/usr/share/milter-manager/python/	\
 	   debian/tmp/usr/share/milter-manager/
 
```

**Comment by daipom** [comment]
The failing tests in focal and jammy are caused by the new specification of `apt`.

* https://blog.jak-linux.org/2020/03/07/apt-2.0/

The latest `apt` can't use wildcards anymore.

File: package/apt/test.sh [hunk 1]
```diff
@@ -22,14 +22,15 @@ sudo apt install -V -y curl lsb-release
 
 code_name=$(lsb_release --codename --short)
 architecture=$(dpkg --print-architecture)
+distribution=$(lsb_release --short --id | tr 'A-Z' 'a-z')
```
Comment (kou) [inline package/apt/test.sh:25]: Distribution, code_name and architecture are normal order such as "Ubuntu Jammy amd64".

```suggestion
distribution=$(lsb_release --short --id | tr 'A-Z' 'a-z')
code_name=$(lsb_release --codename --short)
architecture=$(dpkg --print-architecture)
```

**Comment by daipom** [comment]
> The failing tests in focal and jammy are caused by the new specification of `apt`.
> 
>     * https://blog.jak-linux.org/2020/03/07/apt-2.0/
> 
> 
> The latest `apt` can't use wildcards anymore.

This had nothing to do with the failing tests.

The package path was for debian only. (`${repositories_dir}/debian/pool/...`)
I have fixed it.

File: package/apt/test.sh (hunk 1 above)
Comment (daipom) [reply package/apt/test.sh]: Thanks!

**Comment by daipom** [comment]
Package tests succeed.

**Comment by daipom** [comment]
Thanks for reviewing!

**Pull request merged**